[dagster-airflow] from airflow to dagster guide updates (#11218)
dagster-io/dagster: docs/content/integrations/airflow/from-airflow-to-dagster.mdx

        context.log.info(ds - timedelta(days=7))
```

> Which would yield the following graph of computations in Dagit, Dagster's web-based UI. We'll spin up Dagit later in the tutorial, but wanted to demonstrate :

Reviewer: ```suggestion
Which would yield the following graph of computations in Dagit, Dagster's web-based UI. We'll spin up Dagit later in the tutorial, but wanted to demonstrate:
```

Got an extra space before the colon

  />
</center>

### Op-level retries

> In the tutorial DAG, the `t2` operator allowed for 3 retries. To configure the same behavior in Dagster, you can use [op-level retry policies](/concepts/ops-jobs-graphs/op-retries)

Reviewer: ```suggestion
In the tutorial DAG, the `t2` operator allowed for three retries. To configure the same behavior in Dagster, you can use [op-level retry policies](/concepts/ops-jobs-graphs/op-retries).
```

Missing a period here, and numbers under 10 should be spelled out unless you're referring to a literal value. Ex: "The value of some field is `3`"

---

## Step 2: Define the job

> In Dagster, the computations defined in ops are composed in jobs, which define the sequence and dependency structure of the computations you want to execute. This directly corresponds to a DAG in Airflow. Here, we compose the `op`s `print_date`, `sleep` and `templated` to match the dependency structure defined by the Airflow operators `t1`, `t2`, and `t3`.

Reviewer: ```suggestion
In Dagster, the computations defined in ops are composed in jobs, which define the sequence and dependency structure of the computations you want to execute. This directly corresponds to a DAG in Airflow. Here, we compose the op's `print_date`, `sleep` and `templated` to match the dependency structure defined by the Airflow operators `t1`, `t2`, and `t3`.
```

Can we remove the formatting around **op** here? It looks a bit odd when rendered, and there's no need to use code here

    templated(ds)
```

### Job-level retries

> Job-level retries are managed by the [run launcher](/deployment/run-retries) Once enabled in your dagster.yaml file, you can define the retry count for the job.

Reviewer: ```suggestion
Job-level retries are managed by the [run launcher](/deployment/run-retries). Once enabled in your `dagster.yaml` file, you can define the retry count for the job.
```

Missing a period, and added code formatting to `dagster.yaml`.

dagit -f <your_dagster_file>.py
```

> ## Completed code example

Reviewer: ```suggestion
---

## Completed code example
```

Missing a section divider here

Reviewer: Also, on line 152, which I can't seem to leave a comment on:

> We can now load this file with dagit, the interactive web-ui for dagster

It looks like I missed this in my first review, but there's some capitalization missing. It should be:

> We can now load this file with Dagit, the interactive web-UI for Dagster:
